interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -532,6 +550,9 @@ protected ManagedChannelBuilder<?> createChannelBuilder() {
           channelBuilder.disableServiceConfigLookUp();
           channelBuilder.defaultServiceConfig(serviceConfig);
         }
+        if (TestCases.ORCA.equals(testCase)) {
+          channelBuilder.defaultLoadBalancingPolicy(TEST_ORCA_LB_POLICY_NAME);
```

Comment:
I don't think there's all that clean of a solution to this, but I think I'd prefer having the caller specify the policy via `--service_config_json`. That said, I see how that would prevent running this as a unit test in each transport. That is probably okay; we can make a new junit test class to run it once (with an arbitrary transport).